core/src/components/range/test/range-events.e2e.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('range: events:', () => {
+  test.beforeEach(({ skip }) => {
+    skip.rtl();
+    skip.mode('md');
+  });
+
+  test.describe(' ionChange', () => {
+    test('should not emit if the value is set programmatically', async ({ page }) => {
+      await page.setContent(`<ion-range></ion-range>`);
+
+      const range = page.locator('ion-range');
+      const ionChangeSpy = await page.spyOnEvent('ionChange');
+
+      await range.evaluate((el: HTMLIonRangeElement) => {
+        el.value = 50;
+      });
+
+      await page.waitForChanges();
+
+      expect(ionChangeSpy).toHaveReceivedEventTimes(0);
+
+      // Update the value again to make sure it doesn't emit a second time
+      await range.evaluate((el: HTMLIonRangeElement) => {
+        el.value = 60;
+      });
+
+      await page.waitForChanges();
+
+      expect(ionChangeSpy).toHaveReceivedEventTimes(0);
+    });
+
+    test('should emit when the knob is released', async ({ page }) => {
+      await page.setContent(`<ion-range></ion-range>`);
+
+      const rangeHandle = page.locator('ion-range .range-knob-handle');
+      const ionChangeSpy = await page.spyOnEvent('ionChange');
+
+      const boundingBox = await rangeHandle.boundingBox();
+
+      await rangeHandle.hover();
+      await page.mouse.down();
+      await page.mouse.move(boundingBox!.x + 100, boundingBox!.y);
+
+      await page.mouse.up();
+
+      await ionChangeSpy.next();
+
+      expect(ionChangeSpy).toHaveReceivedEvent();
+      expect(ionChangeSpy).toHaveReceivedEventTimes(1);
```

Comment:
```suggestion
      expect(ionChangeSpy).toHaveReceivedEventTimes(1);
```
I believe the first call is redundant.